packages/slate-tools/slate-tools.config.js

```diff
@@ -35,6 +35,13 @@ module.exports = generate({
         'Enable/disable the prompt to skip uploading settings_data.json',
       default: true,
     },
+    {
+      id: 'providerPlugin',
```

Comment:
Might be a typo but we should keep this consistent with the `providePlugin` plugin name.

Reply:
Good catch! I think "provider" somehow got stuck in my head and I ran with it for everything—including this PR.